Python/Product/Debugger/Debugger.csproj

```diff
@@ -105,6 +105,7 @@
       <Link>Debugger\Transports\WebSocketStream.cs</Link>
     </Compile>
     <Compile Include="AssemblyInfo.cs" />
+    <Compile Include="QualifiedFunctionNameWalker.cs" />
```

Comment:
As far as where this should go - Analysis, perhaps? It feels like a broadly useful thing, and it doesn't have any dependencies.